components/script/dom/window.rs

```diff
@@ -996,7 +996,7 @@ impl Window {
                 page_clip_rect: self.page_clip_rect.get(),
             },
             document: self.Document().upcast::<Node>().to_trusted_node_address(),
-            document_stylesheets: document.stylesheets().clone(),
```

Comment:
Seems to me like we shouldn't build a new vector every time. Maybe we should use an intermediate structure instead of a simple tuple, and just continue to use that when returning from `Document::stylesheets`. What do you think @Ms2ger? Do we really actually care about removing the `Node` bits when using this API?